tsconfig.json

```diff
@@ -1,10 +1,13 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
Happy to use esnext modules as long as they don't cause any build issues downstream

Not sure but this might mean you want to use type=module in package.json if we're going with ESM. Make sure it doesn't break building in nextjs or whatever.

Reply:
Right! I'll test it, @revmischa. 
This change was auto change from the tsc command though, not something I've intentionally changed, but on building the app it got changed. 
In case it breaks something, I'll take a look on it. 
Thanks for the reminder